awesomerc.lua

```diff
@@ -22,6 +22,15 @@ local hotkeys_popup = require("awful.hotkeys_popup")
 -- when client with a matching name is opened:
 require("awful.hotkeys_popup.keys")
 
+local keyname_test = function(key)
```

Comment:
I believe this is test code that got accidentally committed?

Reply:
Shit. How did I send that?